### PR TITLE
Update camera.py to provide width/height

### LINKED ISSUE #11: camera_image() got an unexpected keyword argument 'width'
It looks like there was an API change at some point in HomeAssistant and `width` and `height` need to be added to `camera_image()`
~~~~
homeassistant        | Traceback (most recent call last):                                                                                                             
homeassistant        |   File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request                                          
homeassistant        |     resp = await request_handler(request)                                                                                                      
homeassistant        |   File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle                                                       
homeassistant        |     resp = await handler(request)                                                                                                              
homeassistant        |   File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl                                                  
homeassistant        |     return await handler(request)                                                                                                              
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware                       
homeassistant        |     return await handler(request)                                                                                                              
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 222, in forwarded_middleware                                  
homeassistant        |     return await handler(request)                                                                                                              
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware                       
homeassistant        |     return await handler(request)                                                                                                              
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 219, in auth_middleware                                            
homeassistant        |     return await handler(request)                                                                                                              
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 137, in handle                                                     
homeassistant        |     result = await result                                                                                                                      
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 724, in get                                                 
homeassistant        |     return await self.handle(request, camera)
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 742, in handle                                              
homeassistant        |     image = await _async_get_image(
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 163, in _async_get_image                                    
homeassistant        |     if image_bytes := await camera.async_camera_image(
homeassistant        |   File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 569, in async_camera_image                                  
homeassistant        |     return await self.hass.async_add_executor_job(
homeassistant        |   File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 58, in run                                                               
homeassistant        |     result = self.fn(*self.args, **self.kwargs)
homeassistant        | TypeError: camera_image() got an unexpected keyword argument 'width'                                                                          
~~~~

I was able to get the integration working again with the following changes:
~~~~
diff --git a/camect/camera.py b/camect/camera.py
index 6cc2081..b54b0f3 100644
--- a/camect/camera.py
+++ b/camect/camera.py
@@ -91,7 +91,9 @@ class Camera(camera.Camera):
         """Return a link to the camera feed as entity picture."""
         return None
 
-    def camera_image(self):
+    def camera_image(
+            self, width = None, height = None
+    ):
         """Return a still image response from the camera."""
         return self._home.snapshot_camera(self._device_id)
~~~~

I can put a real PR together if that helps. I figured there might be other changes necessary somewhere.

### PR DESCRIPTION
**Pull request by epleypa**
Update camera.py to fix #11

### CODE REVIEW

File: camect/camera.py [hunk 1]
```diff
@@ -91,7 +91,9 @@ def entity_picture(self):
         """Return a link to the camera feed as entity picture."""
         return None
 
-    def camera_image(self):
+    def camera_image(
+        self, width = None, height = None
+    ):
         """Return a still image response from the camera."""
         return self._home.snapshot_camera(self._device_id)
```
Comment (chao-camect) [inline camect/camera.py:98]: snapshot_camera can take a width and height. Can you make that change?

**Review by chao-camect — COMMENTED** [review]
Sorry I somehow forgot to review this change..

**Comment by pfunkmallone** [comment]
I'll take a look and see if I can figure it out. Thanks

**Comment by chao-camect** [comment]
> I'll take a look and see if I can figure it out. Thanks

Thanks. I mean you can change line 98 to self._home.snapshot_camera(self._device_id, width, height)

**Comment by chao-camect** [comment]
pr/14 is merged. We don't need this one any more. @epleypa @pfunkmallone Thank you!

**Pull request closed without merging**